src/components/Navbar/Navbar.jsx

```diff
@@ -1,13 +1,20 @@
 import React, { Component } from 'react';
 import './Navbar.css';
-
+import slack from './slack.svg';
+import youtube from './youtube.svg';
+import twitter  from './twitter.svg';
+import github from './github.svg';
 class Navbar extends Component {
     render() {
         return (
             <div className="topnav">
+                <img src={slack} className="slack"/>
```

Comment:
Please group the icons and text together and icon should also be clickable.

I'd suggest

```
<a>
  <img/>
  <span>text</span>
</a>
```